client/app/bundles/course/statistics/pages/StatisticsIndex/staff/StaffStatistics.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+
+import { fetchStaffStatistics } from 'course/statistics/operations';
+import LoadingIndicator from 'lib/components/core/LoadingIndicator';
+import Preload from 'lib/components/wrappers/Preload';
+
+import StatisticsTabs from '../StatisticsTabs';
+
+import StaffStatisticsTable from './StaffStatisticsTable';
+
+const StaffStatistics: FC = () => {
+  return (
+    <>
+      <StatisticsTabs />
```

Comment:
Why not create a component <StatisticsContainer /> which is the parent of Student, Staff and Course Stats and place it in the routes with the path: 'statistics'? See how the element in the `path: 'courses/:courseId'` is written

Reply:
done..

I was following the way `UserManagementTabs` is being used, since this component is my reference while implementing this StatisticsTabs. But after looking again, apparently this `UserManagementTabs` component is about to be refactored..